src/zenml/orchestrators/context_utils.py

```diff
@@ -49,3 +51,62 @@ def add_stack_as_metadata_context(
     for k, v in stack_dict.items():
         c_property = context.properties[k]  # type:ignore[attr-defined]
         c_property.field_value.string_value = v
+
+
+def add_schedule_as_metadata_context(
```

Comment:
I couldn't find any usages of this function, is `add_pydantic_object_as_metadata_context` the generic version for it and we can get rid of it entirely?

Reply:
I removed it - it was an artifact of development